applications/matter_weather_station/README.rst

```diff
@@ -12,8 +12,12 @@ Such a device lets you remotely gather different kinds of data using the device
 The device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power, 802.15.4 Thread network.
 You can use this application as a reference for creating your own application.
 
-.. note::
-    The Matter protocol is in an early development stage and must be treated as an :ref:`experimental <software_maturity>` feature.
+Matter weather station product certification
+============================================
+
+The Matter weather station application running on Thingy:53 participated in Matter Specification Validation Event (SVE) and successfully passed all required test cases to be considered as Matter 1.0 compliant device.
+
+See `Matter weather station certification files`_ for more information.
```

Comment:
```suggestion
.. note::
   The Matter weather station application running on Thingy:53 participated in Matter Specification Validation Event (SVE) and successfully passed all required test cases to be considered as Matter 1.0 compliant device.
   See `Matter weather station certification files`_ for more information.
```
I'd just mention it like this, without the specific section. Also, this header level is breaking the build.